sample-app/build.gradle.kts

```diff
@@ -61,6 +62,10 @@ dependencies {
     implementation("androidx.navigation:navigation-fragment:2.5.1")
     implementation("androidx.navigation:navigation-ui:2.5.1")
     implementation(project(":splunk-otel-android"))
+    implementation(project(":splunk-otel-android-volley"))
+    implementation("com.android.volley:volley:1.2.1")
+    implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api:$otelVersion")
+    implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api-semconv:$otelVersion")
```

Comment:
Shouldn't these be included transitively through our projects? Do the users need to add these dependencies too?

Reply:
No, I don't think the main sdk includes these. Volley users will need to include them, yes.